lms/templates/word_cloud.html

```diff
@@ -1,30 +1,46 @@
+<%page expression_filter="h"/>
 <%! from django.utils.translation import ugettext as _ %>
 
-<section
+<div
     id="word_cloud_${element_id}"
     class="${element_class}"
     data-ajax-url="${ajax_url}"
 >
+    % if display_name is not display_name_default:
+    <h3 class="hd hd-3">${display_name}</h3>
+    % endif
+
+    % if instructions is not instructions_default:
+    <div class="input_cloud_section" role="group" aria-labelledby="word_cloud_${element_id}_instructions">
```

Comment:
It's rendering an additional `aria-describedby` if the instructions were set, otherwise it omits it since the referenced `id` would be empty (and would trigger failures).